webpack.config.js

```diff
@@ -5,8 +5,9 @@ module.exports = {
   entry: './src/index.js',
   mode: 'development',
   output: {
-    filename: 'bundle.js',
+    filename: 'index.bundle.js',
     path: path.resolve(__dirname, 'dist'),
+    clean: true,
   },
   plugins: [
     new HtmlWebpackPlugin({
```

Comment:
- [ ] You need to configure the basic commands for building Webpack in development and production mode using the `package.json` file. I believe you've done something similar in your Webpack exercise. We should be able to set up and run Webpack devserver with no errors, using the configuration. At the moment, package.json is completely missing in your project. Kindly fix this.